docs/source/getting-started-security.md

```diff
@@ -79,9 +79,10 @@ The recommended approach for performing user impersonation when the kernel is la
 is to configure the `spark-submit` command within 
 [run.sh](https://github.com/jupyter-incubator/enterprise_gateway/blob/master/etc/kernelspecs/spark_python_yarn_cluster/bin/run.sh)
 to use the `--proxy-user ${KERNEL_USERNAME}` option.  This YARN option  requires that kerberos be configured within the
-cluster.  Please refer to the
+cluster.  In addition the gateway user (`elyra` by default) needs to be set up as a `proxyuser` in hadoop configs.
```

Comment:
When reading the doc reference, it seems like the docs refer to this as a *superuser*.  As a result, I think it might be more clear to say 
>... needs to be set up as a `hadoop.proxyuser` *superuser* in hadoop configs.

IMO, this makes `proxyuser` more distinct from `--proxy-user` since the two are only a typo away from each other but mean very different things.